style: Cleanup ERTP Guide (#660)
Agoric/documentation: main/ertp/guide/amounts.md

![Amount methods](./assets/amount.svg) 

`Amounts` describe digital assets. There are no `amount` API methods.

> Instead, an `amount` has `brand` and `value` properties and other API methods
> take them as arguments.

Reviewer: ```suggestion
Instead, an `amount` has `brand` and `value` properties. AmountMath methods
take amounts as arguments.
```
